src/main/resources/application.conf

```diff
@@ -242,6 +242,9 @@ mantis {
     # If false then that 20% gets burned
     # Doesn't have any effect is ecip1098 is not yet activated
     treasury-opt-out = false
+
+    # List of hex encoded public keys of Checkpoint Authorities
+    checkpoint-public-keys = []
```

Comment:
Shouldn't this be chain specific and go on the chains/ files?

Reply:
Do you mean the genesis file? While that sounds like a good idea, we've been talking about modifying the set of public key in order exclude a potential rogue node. Until we have a clear approach I would leave as it is.

In general I would avoid any deviations from our current reference implementation, until we have concrete improvement proposals.

Reply:
he means chain specific configuration files (resources/chains/etc.conf)

Reply:
Didn't realise that existed. I like that 👍